packages/sdks/src/blocks/tabs/component-info.ts

```diff
@@ -0,0 +1,152 @@
+const defaultTab = {
+  '@type': '@builder.io/sdk:Element',
+  responsiveStyles: {
+    large: {
+      paddingLeft: '20px',
+      paddingRight: '20px',
+      paddingTop: '10px',
+      paddingBottom: '10px',
+      minWidth: '100px',
+      textAlign: 'center',
+      display: 'flex',
+      flexDirection: 'column',
+      cursor: 'pointer',
+      userSelect: 'none',
+    },
+  },
+  component: {
+    name: 'Text',
+    options: {
+      text: 'New tab',
+    },
+  },
+};
+
+const defaultElement = {
+  '@type': '@builder.io/sdk:Element',
+  responsiveStyles: {
+    large: {
+      height: '200px',
+      display: 'flex',
+      marginTop: '20px',
+      flexDirection: 'column',
+    },
+  },
+  component: {
+    name: 'Text',
+    options: {
+      text: 'New tab content ',
+    },
+  },
+};
+
+export const componentInfo: any = {
```

Comment:
can we use the `ComponentInfo` type here?

Reply:
yes! thanks!!